packages/webapp/src/pages/Connection/Authorization.tsx

```diff
@@ -16,6 +17,15 @@ interface AuthorizationProps {
 
 export default function Authorization(props: AuthorizationProps) {
     const { connection, forceRefresh, loaded } = props;
+    const [refreshing, setRefreshing] = useState(false);
+
+    useEffect(() => {}, [refreshing]);
```

Comment:
What's the use of this useEffect? ☺️

Reply:
You are right, thanks for catching that, useEffect has no effect here ☺️. I was actually thinking instead of having refreshing in the SecretInput we can just change Loading icon to a spinner . What do you think?

Reply:
imo I think it's better yes but I'm not the one deciding here so I can't say for sure :D